pkg/validation-webhook/snapshot.go

```diff
@@ -43,12 +45,13 @@ type SnapshotAdmitter interface {
 }
 
 type admitter struct {
-	lister storagelisters.VolumeSnapshotClassLister
+	snapshotLister      snapshotListers.VolumeSnapshotClassLister
+	groupSnapshotLister groupSnapshotListers.VolumeGroupSnapshotClassLister
 }
 
-func NewSnapshotAdmitter(lister storagelisters.VolumeSnapshotClassLister) SnapshotAdmitter {
+func NewSnapshotAdmitter(lister snapshotListers.VolumeSnapshotClassLister) SnapshotAdmitter {
 	return &admitter{
-		lister: lister,
+		snapshotLister: lister,
```

Comment:
does this also need to initialise the group snapshot lister?

Reply:
> does this also need to initialise the group snapshot lister?

refactored a bit, This struct is not shared with group snapshot now.